Silicon/NXP/iMX6Pkg/Library/iMX6BoardLib/iMX6BoardHelper.masm

```diff
@@ -0,0 +1,65 @@
+/** @file

```

Comment:
I don't think we ever compiled those .masm files, no idea if they will work or not.

Reply:
We only support GCC per our DSC files. So we are up-streaming code we didn't test. Anyone who looks into our DSC file will figure that out.

Reply:
fixed